sdk/python/touca/cli/_post.py

```diff
@@ -45,10 +45,14 @@ def parser(self, parser: ArgumentParser):
         parser.add_argument("src", help="path to directory with binary files")
         parser.add_argument("--api-key", help="Touca API Key", dest="api-key")
         parser.add_argument("--api-url", help="Touca API URL", dest="api-url")
+        parser.add_argument(
+            "--dry-run", 
+            action="store_true", 
+            help="See what files would be posted",
+            dest="dry-run" )
```

Comment:
It's great that you've added this option to the `post` request (since other sub-commands don't support this option yet) but I think at some point we should consider moving it to the top-level and extending support for all sorts of sub-commands.

Reply:
Where specifically can I find the top-level for each sub-command?

Reply:
It is in `__main__`. I'd say we should leave this as is for now. My comments was just for the record.